pkgs/servers/http/nginx/modules.nix

```diff
@@ -248,6 +266,7 @@ in
   };
 
   modsecurity-nginx = {
+    name = "modsecurity";
```

Comment:
Same name as `modsecurity` since this is an older version of the same module. Maybe it should be deprecated.

Reply:
Official package name - ModSecurity v3 Nginx Connector. Can be renamed to modsecurity-nginx.
In version 3.x, the ModSecurity package has been completely redesigned. In version 3.x ModSecurity has been completely redesigned into library - libmodsecurity. And added apache and nginx connectors.
That's why we have two different modules now.

Reply:
Perhaps you should not use two modules at the same time. There may be conflicts, I haven't checked.

Reply:
I'd say if both modules are included it's more likely to be an accident. If one somehow really wants that they can just use `modsecurity-nginx // { name = "modsecurity-nginx"; }`.

Reply:
Also FYI nginx doesn't seem to build with `modsecurity`:

```
/nix/store/gsk4ig21jd7q5q0sa8dlrn9lm64jlck1-modsecurity-2.9.6-nginx/nginx/modsecurity/ngx_http_modsecurity.c: In function 'ngx_http_modsecurity_save_headers_in':
/nix/store/gsk4ig21jd7q5q0sa8dlrn9lm64jlck1-modsecurity-2.9.6-nginx/nginx/modsecurity/ngx_http_modsecurity.c:382:39: error: 'ngx_http_headers_in_t' has no member named 'cookies'; did you mean 'cookie'?
  382 |     if (ngx_array_init(&r->headers_in.cookies, r->pool, 2,
      |                                       ^~~~~~~
      |                                       cookie
In file included from /nix/store/gsk4ig21jd7q5q0sa8dlrn9lm64jlck1-modsecurity-2.9.6-nginx/nginx/modsecurity/../../standalone/api.h:23,
                 from /nix/store/gsk4ig21jd7q5q0sa8dlrn9lm64jlck1-modsecurity-2.9.6-nginx/nginx/modsecurity/ngx_http_modsecurity.c:25:
At top level:
/nix/store/gsk4ig21jd7q5q0sa8dlrn9lm64jlck1-modsecurity-2.9.6-nginx/nginx/modsecurity/../../apache2/modsecurity.h:138:13: error: 'geo_lock_name' defined but not used [-Werror=unused-variable]
  138 | static char geo_lock_name[L_tmpnam];
      |             ^~~~~~~~~~~~~
/nix/store/gsk4ig21jd7q5q0sa8dlrn9lm64jlck1-modsecurity-2.9.6-nginx/nginx/modsecurity/../../apache2/modsecurity.h:137:13: error: 'auditlog_lock_name' defined but not used [-Werror=unused-variable]
  137 | static char auditlog_lock_name[L_tmpnam];
      |             ^~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```

Reply:
https://github.com/SpiderLabs/ModSecurity/issues/2768 - ModSecurity v2 with nginx is not a supported combination.

Reply:
Should we just remove it and make `modsecurity` and `modsecurity-nginx` aliases to the same thing?

Reply:
Just remove the modsecurity module and leave modsecurity-nginx unchanged?
The connectors in ModSecurity have the analogs names - ModSecurity-nginx and ModSecurity-apache.